packages/core/tracing/src/partials/SummaryViewTab.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <Splitpanes
+    class="summary-view"
+    horizontal
+  >
+    <Pane
+      class="lifecycle-pane"
+      min-size="20"
+    >
+      <div class="lifecycle-view-wrapper">
+        <LifecycleView
+          v-if="rootSpan"
+          :root-span="rootSpan"
+          :show-skeleton="showSkeleton"
+        />
+      </div>
+    </Pane>
+    <Pane
+      class="detail-pane"
+      size="50"
+    >
+      <template v-if="rootSpan">
+        <RequestInfo
+          :root-span="rootSpan"
+          :show-skeleton="showSkeleton"
+        />
+
+        <PayloadDisplay
+          v-if="payloads?.headers?.request"
```

Comment:
Optional chaining doesn't break GitHub's syntax highlight anymore?

Reply:
It seems to be so that I didn't even notice that… 🥹